app/controllers/api/contacts_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::ContactsController < ApplicationController
+  def index
```

Comment:
Why are you sending different keys in the response for the similar reply from backend ?